curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zhang-suen-thinning-algorithm.english.md

```diff
@@ -44,49 +44,51 @@ It produces the thinned output:
      #                             ############
                        ###                          ###
 
-                                                           </pre>
+</pre>
+
 <h2>Algorithm</h2>
 Assume black pixels are one and white pixels zero, and that the input image is a rectangular N by M array of ones and zeroes.
 The algorithm operates on all black pixels P1 that can have eight neighbours. The neighbours are, in order, arranged as:
-<table border="1">
-  <tr><td>P9</td><td>P2</td><td>P3</td></tr>
-  <tr><td>P8</td><td><b>P1</b></td><td>P4</td></tr>
-  <tr><td>P7</td><td>P6</td><td>P5</td></tr>
-</table>
-Obviously the boundary pixels of the image cannot have the full eight neighbours.
 
-    Define $A(P1)$ = the number of transitions from white to black, (0 -> 1) in the sequence P2,P3,P4,P5,P6,P7,P8,P9,P2. (Note the extra P2 at the end - it is circular).
+| P9 | P2 | P3 |
```

Comment:
Unfortunately I think this might be an instance where it's better to stick with the original table so that just the center element is bold. Here's one that looks closer to how it appears on Rosetta Code:

```
<table border="3">	
  <tr><td style="text-align: center;">P9</td><td style="text-align: center;">P2</td><td style="text-align: center;">P3</td></tr>	
  <tr><td style="text-align: center;">P8</td><td style="text-align: center;"><b>P1</b></td><td style="text-align: center;">P4</td></tr>	
  <tr><td style="text-align: center;">P7</td><td style="text-align: center;">P6</td><td style="text-align: center;">P5</td></tr>	
</table>
```